state/state.go

```diff
@@ -150,6 +153,13 @@ type SetValue struct {
 	Values []string `yaml:"values"`
 }
 
+// AffectedReleases hold the list of released that where updated, deleted, or in error
+type AffectedReleases struct {
+	Upgraded []*ReleaseSpec
+	Deleted  []*ReleaseSpec
+	Error    []*ReleaseSpec
```

Comment:
Better use Errored, or Failed for consistency?

Reply:
yes good idea, I'll try to make another PR with your comments